src/Streamly/Data/Fold.hs

```diff
@@ -65,14 +66,17 @@
 -- >>> Stream.fold f stream
 -- ("Even 50","Odd 50")
 --
--- Terminating folds can be combined to parse the stream serially such that the
```

Comment:
We should probably introduce the term `Terminating folds` before use. In this case, we can just use `Folds` I guess.